alerts/UA00002.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dateRaised": 20201028,
+  "affectedFirmware": ["all"],
+  "hardwareLimited": [],
+  "description": "When using SBUS output, if the RC receiver is lost the IOMCU will invert the SBUS output momentarily whilst searching for an FPort input. This can lead to a loss of signal on the SBUS output port. This only affects boards with an IOMCU",
```

Comment:
Maybe we could simplify this by prefixing, "SBUS output stutters if the the RC receiver stops sending signals which may happen during an RC Failsafe"